src/modules/LanUdevAuto.rb

```diff
@@ -194,39 +198,43 @@ def Import(settings)
     end
 
     def Write
-      comment = [
-        "# Generated by autoyast",
-        "# program run by the persistent-net-generator.rules rules file.",
-        "#",
-        "# You can modify it, as long as you keep each rule on a single line."
-      ]
+      template = "SUBSYSTEM==\"net\", ACTION==\"add\", DRIVERS==\"?*\", %s==\"%s\", NAME=\"%s\""
```

Comment:
this looks like constant. And I also think that with so many replacement it is better to use `%{name}` syntax, so it will be easier to read it